layouts/includes/footer.cfm

```diff
@@ -9,7 +9,7 @@
 		manifestRoot : args.manifestRoot
 	)#"></script>
 	<script async="async" defer="defer" src="#event.getModuleRoot( "cbDebugger" )##html.elixirPath(
-		fileName : "js/App.js",
+		fileName : "js/Cbdebugger.js",
```

Comment:
can you just use `cbdebugger.js` all lowercase.

Reply:
Agreed. I like lowercase as well.